pom.xml

```diff
@@ -521,6 +539,8 @@
         <mavan.findbugsplugin.exclude.file>findbugs-exclude-filter.xml</mavan.findbugsplugin.exclude.file>
         <carbon.kernel.version>4.6.1-m5</carbon.kernel.version>
         <carbon.multitenancy.version>4.8.5</carbon.multitenancy.version>
+        <carbon.identity.media.repository.version>1.0.4-SNAPSHOT</carbon.identity.media.repository.version>
```

Comment:
update this to a release version

Reply:
need to merge https://github.com/wso2/identity-content-repository/pull/7 for this.